### PR TITLE
docs: Add ROADMAP + CONTRIBUTING

### PR DESCRIPTION
**Pull request by lukehesluke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This repository hosts two different projects:
 * [Broker microservice](./packages/openactive-broker-microservice/): this sits in between the test suite and the target Open Booking API implementation. This allows the test suite to watch for changes, and throws them back to it.
 * [Integration tests](./packages/openactive-integration-tests): this performs automated tests against the API
 
+[The Roadmap](./ROADMAP.md) provides an overview of upcoming milestones and when they're expected to be completed.
```
Comment (nickevansuk) [inline README.md:11]: Perhaps:

"provides an overview of upcoming development milestones, and contributions are welcome"

To put the focus on community involvement?

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Milestones
+
+This doc is our roadmap. It's a place to start to investigate the myriad of issues that you can contribute to. Check out the different milestones listed below, but feel free to explore the issues by label too.
+
+Please check out our [contributing guidelines](./CONTRIBUTING.md) and code of conduct to help you get started, and the [README](./README.md) for an overview if you haven't read it yet!
+
+### Milestone 1: [“core”](https://github.com/openactive/openactive-test-suite/tree/master/packages/openactive-integration-tests/test/features/core) and [“payment”](https://github.com/openactive/openactive-test-suite/tree/master/packages/openactive-integration-tests/test/features/payment) features
+
+_Expected 12th August_
```
Comment (nickevansuk) [inline ROADMAP.md:13]: Perhaps:

_Aiming for completion by 12th August_

More inclusive language for those contributing?

File: README.md (hunk 1 above)
Comment (lukehesluke) [reply README.md]: Done

File: ROADMAP.md (hunk 2 above)
Comment (lukehesluke) [reply ROADMAP.md]: Done

**Pull request merged**